lib/types.d.ts

```diff
@@ -95,13 +95,19 @@ export interface ClassDef {
   decorators?: DecoratorDef[];
 }
 
+export type ClassConstructorParamDef = ParamDef & {
```

Comment:
Nitpick: seems more consistent to use an interface?

Reply:
You can't here, because `ParamDef` is a tagged union type alias and interfaces can't extend type aliases.